wagtailio/project_styleguide/templates/patterns/components/streamfields/cards/cards_list_block.html

```diff
@@ -1,5 +1,5 @@
-{% for card in value.cards %}
-    <div>
+<div class="card-grid">
```

Comment:
Suggestion: Wrap this in `<div class="grid">` so it won't be full width when used on a page template. The following css for `.card-grid` should align it across the breakpoints:

```
.card-grid {
    grid-column: 2 / span 2;

    @include media-query(medium) {
        margin-bottom: 50px;
        grid-column: 2 / span 3;
    }

    @include media-query(large) {
        grid-column: 2 / span 5;
    }
```